edu.cuny.citytech.defaultrefactoring.eval/src/edu/cuny/citytech/defaultrefactoring/eval/handlers/FindCandidateSkeletalImplementationsHandler.java

```diff
@@ -51,21 +56,25 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 			FileWriter interfacesWriter = new FileWriter("interfaces.csv");
 			FileWriter classesImplementingInterfacesWriter = new FileWriter("classes_implementing_interfaces.csv");
 			FileWriter classesExtendingClassesWriter = new FileWriter("classes_extending_classes.csv");
-
+			FileWriter methodsWriter = new FileWriter("methods.csv");
+			
+			
 			// getting the csv file header
 			String[] typesHeader = { "Project Name", ",", "CompilationUnit", ",", "Fully Qualified Name" };
 			String[] classesHeader = { "Fully Qualified Name" };
 			String[] abstractClassesHeader = { "Fully Qualified Name" };
 			String[] interfacesHeader = { "Fully Qualified Name" };
 			String[] classesImplementing_interfacesHeder = { "Class FQN", ",", "Interface FQN" };
 			String[] classesExtendingClassesHeader = { "Source Class FQN", ",", "Target Class FQN" };
-
+			String[] methodsHeader = { "Method Identifier", "Declaring Type FQN"};
+			CSVPrinter metPrinter = new CSVPrinter(methodsWriter, CSVFormat.EXCEL.withHeader(methodsHeader));
+			
 			csvHeader(typesWriter, typesHeader);
 			csvHeader(classesWriter, classesHeader);
 			csvHeader(abstractClassesWriter, abstractClassesHeader);
 			csvHeader(interfacesWriter, interfacesHeader);
 			csvHeader(classesImplementingInterfacesWriter, classesImplementing_interfacesHeder);
-			csvHeader(classesExtendingClassesWriter, classesExtendingClassesHeader);
+			csvHeader(classesExtendingClassesWriter, classesExtendingClassesHeader);			
```

Comment:
This looks like a white space change. Can we remove it? Doesn't seem relevant to the task.